.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         jira-version: ["7.2.0", "7.13.0", "8.0.0", "8.5.0", "8.13.0"]
```

Comment:
Do we need all the old versions? Maybe we could start with 8.13 and add 8.20, 9.3

Reply:
Maybe we don't need them, however other changes in this PR don't need those to be changed, so I will leave them as they are.

Reply:
I actually could add 8.20.0 and 9.0.0 to reveal compatibility